src/loaders/ncbi_downloader/ncbi_downloader.py

```diff
@@ -0,0 +1,73 @@
+"""
```

Comment:
Hmm, I wouldn't expect to find a downloader in loaders.. do we need a new directory?

Reply:
maybe fine for now. Because that's the downloader we need for the loaders.

Reply:
Fair enough